candidates/tests/auth.py

```diff
@@ -23,11 +23,14 @@ def setUpClass(cls):
                 ('ermintrude', 'user_who_can_rename', [TRUSTED_TO_RENAME_GROUP_NAME]),
                 ('frankie', 'user_who_can_record_results', [RESULT_RECORDERS_GROUP_NAME]),
         ):
-            u = User.objects.create_user(
-                username,
-                username + '@example.com',
-                'notagoodpassword',
-            )
+            try:
+                u = User.objects.get(username=username)
+            except User.DoesNotExist:
```

Comment:
I'm having some trouble reproducing a situation in which this is needed, since the `tearDownClass` method seems to remove them whatever I do! Could you suggest a way of reproducing the problem where they're left over?